src/main/kotlin/org/wycliffeassociates/otter/jvm/app/theme/AppStyles.kt

```diff
@@ -0,0 +1,89 @@
+package org.wycliffeassociates.otter.jvm.app.theme
+
+import de.jensd.fx.glyphs.materialicons.MaterialIcon
+import de.jensd.fx.glyphs.materialicons.MaterialIconView
+import javafx.scene.paint.Color
+import javafx.scene.text.Font
+import javafx.scene.text.FontWeight
+import org.wycliffeassociates.otter.jvm.app.widgets.progressdialog.ProgressDialogStyles
+import tornadofx.*
+
+class AppStyles : Stylesheet() {
+
+    companion object {
+        val backButton by cssclass()
+        val appBackground by cssclass()
+        val progressDialog by cssclass()
+
+        // Icons
+        fun recordIcon(size: String = "1em") = MaterialIconView(MaterialIcon.MIC_NONE, size)
+        fun editIcon(size: String = "1em") = MaterialIconView(MaterialIcon.EDIT, size)
+        fun viewTakesIcon(size: String = "1em") = MaterialIconView(MaterialIcon.APPS, size)
+        fun backIcon() = MaterialIconView(MaterialIcon.ARROW_BACK)
+        fun chapterIcon(size: String = "1em") = MaterialIconView(MaterialIcon.CHROME_READER_MODE, size)
```

Comment:
MaterialIconView is a FontAwesome class, so it uses css dimensions to specify the size of the icon. I'm using "1em" here since that is the default size MaterialIconView uses internally if no size string is passed in